parser-typechecker/src/Unison/Codebase/Metadata.hs

```diff
@@ -9,15 +9,15 @@ where
 
 import Data.Map qualified as Map
 import Unison.Prelude
-import Unison.Reference (Reference)
+import Unison.Reference (TermReference)
 import Unison.Util.List qualified as List
 import Unison.Util.Relation qualified as R
 import Unison.Util.Star3 (Star3)
 import Unison.Util.Star3 qualified as Star3
 
-type Type = Reference
+type Type = () -- dummy value, intermediate phase of removing metadata altogether
```

Comment:
I'm a little confused, what's the use of keeping a dummy type here? Is this being used anywhere that we can't also just delete?

Reply:
I think we'd need to write a `Star2` type and drop the `Type` index here. It would be better, but it just felt like busywork that was already slated to be thrown out.

Reply:
I got nerdsniped

Reply:
an hour later and no end in sight... it doesn't help that HLS lied about what's safe to delete, when it hadn't loaded all the packages yet to know.

Reply:
Alternative at https://github.com/unisonweb/unison/compare/topic/NeedTypeForBuiltinMetadata-more
I agree it is better, but in retrospect also agree with past self that it's hard to justify the time expenditure when it's going to be tossed anyway.